internal/scan/scan.go

```diff
@@ -13,19 +13,22 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
-	ERC721 "github.com/freeverseio/laos-universal-node/internal/platform/blockchain/erc721"
+	"github.com/freeverseio/laos-universal-node/internal/platform/blockchain/ERC721BridgelessMinting"
 )
 
 var (
-	eventTransferName          = "Transfer"
-	eventApprovalName          = "Approval"
-	eventApprovalForAllName    = "ApprovalForAll"
-	eventTransferSig           = []byte(fmt.Sprintf("%s(address,address,uint256)", eventTransferName))
-	eventApprovalSig           = []byte(fmt.Sprintf("%s(address,address,uint256)", eventApprovalName))
-	eventApprovalForAllSig     = []byte(fmt.Sprintf("%s(address,address,bool)", eventApprovalForAllName))
-	eventTransferSigHash       = crypto.Keccak256Hash(eventTransferSig).Hex()
-	eventApprovalSigHash       = crypto.Keccak256Hash(eventApprovalSig).Hex()
-	eventApprovalForAllSigHash = crypto.Keccak256Hash(eventApprovalForAllSig).Hex()
+	eventTransferName                      = "Transfer"
+	eventApprovalName                      = "Approval"
+	eventApprovalForAllName                = "ApprovalForAll"
+	eventNewERC721BridgelessMintingName    = "NewERC721BridgelessMinting"
```

Comment:
so when we use new version of contract we have to change these names right? does it make sense to change that in this PR?

Reply:
These names have already been updated, you may be seeing an outdated version of the PR. Please pull the changes to see the updated version